src/templates/offer.html

```diff
@@ -10,29 +10,37 @@
 <body>
 {%FILE=header.snippet.html%}
 	<div class="body">
-		<h1>Your Offer</h1>
+		<h1>Your Donation Swap Offer</h1>
 		<table id="table">
 			<tbody>
+				<tr>
+					<td>Your Name:</td>
+					<td><span id="name"></span>
+				</tr>
+				<tr>
+					<td>Charity you wish to support:</td>
+					<td><span id="charity"></span></td>
+				</tr>
 				<tr>
 					<td>Amount:</td>
 					<td><span id="currency"></span> <span id="amount"></span></td>
 				</tr>
 				<tr>
-					<td>Charity:</td>
-					<td><span id="charity"></span></td>
+					<td>Minimum swap amount:</td>
+					<td><span id="currency"></span> <span id="minamount"></span></td>
```

Comment:
can you please add `<!-- xxx TODO support minamount -->` in there, so I don't forget?